common-content/en/module/complexity/caching/index.md

```diff
@@ -1,11 +1,11 @@
 +++
 title = "Caching"
+time = 15
```

Comment:
Is this why the time is wrong on the prep?? The order?

Reply:
Yeah, before the `time` properties were getting treated as `build.time` (because they were nested in that section) rather than top-level `time` properties.

(I am such a fan of build errors when you set a property that isn't used...!)